test/MorseL.Tests/EndToEndTests.cs

```diff
@@ -191,7 +191,7 @@ public async void HubInvokingNonExistentClientMethodThrowsInHub(string methodNam
             }
         }
 
-        [Theory]
+        [Theory(Skip = "Flaky on Linux and macOS. Seems reliable on Windows.")]
```

Comment:
Same as above? Treat it as expected to succeed under windows, expected to fail under *nix?

Reply:
Regrettably it sometimes succeeds and sometimes fails on non-Windows platforms. For now, I'll leave it skipped and get that removed this weekend with work on #48